<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>83</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(147, 140, 147);
color: rgb(2, 2, 2);</string>
    </property>
    <property name="placeholderText">
     <string>Recipe#</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>discharge</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 7, 19);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>test</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>mix</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QChartView" name="phChartView" native="true">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>90</y>
      <width>221</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QChartView" name="po4ChartView" native="true">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>260</y>
      <width>221</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QChartView" name="clChartView" native="true">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>440</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>show setting</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>881</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">QtCharts/QChartView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
